Screens/Cart/cart.css

```diff
@@ -0,0 +1,106 @@
+.cart-container {
+  margin: 5rem 5rem 0rem;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.cart-container > .cart-list {
+  flex: 0.6;
+  padding: 3rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: flex-end;
```

Comment:
can make separate class for this in main css file

Reply:
Thank for pointing it out. On it